src/test/offline/io/stargate/sgv2/jsonapi/service/operation/model/impl/OfflineGetStatusOperationTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import io.quarkus.test.junit.QuarkusTest;
```

Comment:
doesn't seem in scope of this PR. Kathir is refactoring these classes to another repo anyway.